litellm/llms/azure/chat/gpt_transformation.py

```diff
@@ -121,11 +121,11 @@ def _is_response_format_supported_api_version(
     ) -> bool:
         """
         - check if api_version is supported for response_format
+        https://learn.microsoft.com/en-us/azure/ai-services/openai/how-to/structured-outputs?tabs=python-secure%2Cdotnet-entra-id&pivots=programming-language-csharp#api-support
         """
-
-        is_supported = int(api_version_year) <= 2024 and int(api_version_month) >= 8
-
-        return is_supported
+        year = int(api_version_year)
+        month = int(api_version_month)
+        return (year == 2024 and month >= 8) or year > 2024
```

Comment:
nit: tuples have a columnar comparison, so could be simplified for readability

```python
(year, month) >= (2024, 8)
```

examples:
```
>>> (2024, 8) >= (2024, 8)
True
>>> (2024, 7) >= (2024, 8)
False
>>> (2025, 7) >= (2024, 8)
True
>>> (2023, 9) >= (2024, 8)
False
>>> (2024, 9) >= (2024, 8)
True
```

Reply:
I prefer the current approach. Easier to read